### PR TITLE
Add testcase2 scripts

### LINKED ISSUE #448: fault tolerant + autoscale实验方法需要更新
实验的目标：证明开启autoscale的集群，多种场景下在以下指标优于没有autoscale的集群

- 集群整体平均资源利用率（越大越好）
- 平均每个job的执行时间（越短越好）
- 平均每个job的等待时间，从提交到开始执行（越短越好）

实验变量：是否开启autoscale(通过是否提交trainingjob resource确定)

实验不变量：
- pserver 个数
- pserver CPU
- 期望的trainer并发数
- trainer CPU
- job程序

### PR DESCRIPTION
**Pull request by Yancey1989**
Fixed #448

### CODE REVIEW

File: doc/autoscale/experiment/README.md [hunk 1]
```diff
@@ -20,76 +20,83 @@
...
-- Submit the general training job 
-    1. Submit a job(job-A), which requests 100 trainer Pods(1 CPU cores per Pod), 2 pservers and 1 master.
-    1. Submit another job(job-B), which requests 200 trainer Pods(1 CPU cores per Pod), 2 pservers and 1 master.
-    1. The job-B will be the pending status only if job-A finished because there are not enough CPU cores for the requests.
-- Submit the auto-scaling training job
-    1. Submit a job(job-A), which requests 100 trainer Pods(1 CPU core per Pod, min-instances is 50, max-instances is 500), 2 pservers and 1 master, And then job-A will be scaled up to immediately to use the maximum free resources(max 500 trainer Pods).
-    1. Submit another job(job-B), which requests 200 trainer Pods(1 CPU core per Pod, min-instances is 50, max-instances is 200), 2 pservers and 1 master.
-    1. Job-A will be scaled down and job-A and job-B will run in the cluster at the same time, and they will use the maximum free resources.
+- Submit the fault-tolerant training job 
+    1. Submit 10 jobs, which have 20 parallelisms(requests 10 CPU per Pod), 10 pservers and 1 master.
```
Comment (typhoonzero) [inline doc/autoscale/experiment/README.md]: which have 20 parallelisms => which request 21 cores of CPU resource.

File: doc/autoscale/experiment/README.md [hunk 2]
```diff
@@ -20,76 +20,83 @@
...
+    1. The resource which the jobs requested need greater than the resource of the cluster.
```
Comment (typhoonzero) [inline doc/autoscale/experiment/README.md]: need greater than => need to be greater than

File: doc/autoscale/experiment/README.md [hunk 3]
```diff
@@ -20,76 +20,83 @@
...
+    1. Collect the time series data and the statistical result.
+- Submit the fault-tolerant training job with auto-scaling
+    1. Submit 10 jobs, which have 5 parallelisms(request 10 CPU per Pod), 10 pservers and 1 master.
```
Comment (typhoonzero) [inline doc/autoscale/experiment/README.md]: Same as above, change to CPU resource requests.

Comment (typhoonzero) [reply doc/autoscale/experiment/README.md]: And, maybe we need to add: assume memory request is always satisfied.

File: doc/autoscale/experiment/README.md [hunk 4]
```diff
@@ -20,76 +20,83 @@
...
+    1. Submit the TrainingJob, which have min-instance is 2 and max-instance is 20.
```
Comment (typhoonzero) [inline doc/autoscale/experiment/README.md]: TrainingJob(Kubernetes thirdpartyresource)

File: k8s/controller/controller.yaml [hunk 5]
```diff
@@ -11,6 +12,8 @@ spec:
     spec:
       containers:
       - name: training-job-controller
-        image: helinwang/training-job-controller
+        image: registry.baidu.com/paddlepaddle/controller:helin
```
Comment (Yancey1989) [inline k8s/controller/controller.yaml]: Not a general image, just use this for test.

Comment (helinwang) [reply k8s/controller/controller.yaml]: Curious is this has the same content as `helinwang/training-job-controller`? (otherwise what has been added?)

Comment (Yancey1989) [reply k8s/controller/controller.yaml]: Yes, the same as `helinwang/training-job-controller`, the internal CPU cluster use the internal registry server will be faster then the Docker Hub.

File: doc/autoscale/experiment/README.md [hunk 6]
```diff
@@ -143,33 +143,61 @@ serving together. This case is a very typical scenario for large enterprises and
...
+        ```bash
+        > JOB_COUNT=5 ./run.sh start case2
+        ```
+	1. Get the time series data.
+		The time serise data will be appended in the file `./out/mnist-case[1|2]-pass[0-9].log`
+		as the following format:
+
+
+		```
+		0,2.11,0,3,0,0,0,0,0|0|0,0.00|0.00|0.00
```
Comment (putcn) [inline doc/autoscale/experiment/README.md]: I saw duplicated timestamps from different line of records from @helinwang 's change, but here is no more. was this a bug?

**Pull request merged**